src/lib.rs

```diff
@@ -130,7 +130,7 @@ impl Default for CodeEditor {
             rows: 10,
             vscroll: true,
             stick_to_bottom: false,
-            shrink: false,
+            desired_width: f32::INFINITY,
```

Comment:
As far as I understand `f32::INFINITY` is the value egui documents for "Use max space" behavior.
So I replaced `f32::MAX` with it.

If we need to use `f32::MAX` here instead for whatever reason, we should probably leave a comment explaining why.